airbyte-webapp/src/components/connectorBuilder/Builder/AddStreamButton.module.scss

```diff
@@ -0,0 +1,20 @@
+@use "scss/variables";
+@use "scss/colors";
+
+$buttonWidth: 26px;
+
+.body {
+  display: flex;
+  flex-direction: column;
+  gap: variables.$spacing-xl;
+}
+
+.addButton {
```

Comment:
I think it's OK for now, but I don't like how much you need to fight the stylings of the underlying button component.

It seems like it would be possible to make the round icon button a prop on the regular button - this way you don't need to overrule styles with `!important` but you can selectively apply the right styles in the right moment.

If you think that makes sense I can create an issue for it.

Reply:
Yeah I think that makes sense - having a prop on the button to easily make it round would be really nice here. If you could create an issue for that, that would be great!